.mise/tasks/infrastructure/down

```diff
@@ -1,4 +1,8 @@
 #!/usr/bin/env sh
 
-# mise description="destroy infrastructure"
-# mise depends=["infrastructure:core:destroy", "infrastructure:kubeflow:destroy"]
+# mise description="launch infrastructure"
```

Comment:
Typo: should be "destroy" or "teardown" or something.

Reply:
ah good catch, fixed